packages/google_sign_in/google_sign_in_web/pubspec.yaml

```diff
@@ -22,9 +22,14 @@ dependencies:
     sdk: flutter
   flutter_web_plugins:
     sdk: flutter
-  google_sign_in_platform_interface: ^2.0.0
+  google_sign_in_platform_interface: ^2.2.0
```

Comment:
I like this version bump, however this makes it necessary to land and publish google_sign_in_platform_interface first.

(I'm not against that either, just restating what probably has been mentioned before!